dist/init/linux-systemd/README.md

```diff
@@ -6,26 +6,153 @@ if you have any questions. Feel free to prepend to your question
 the username of whoever touched the file most recently, for example
 `@wmark re systemd: …`.
 
-The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The provided file should work with systemd version 229 or later. It might work with earlier versions.
```

Comment:
It will not work with earlier versions. Better strip the second sentence here, because anything that adds to the first would be redundant.